src/rust/bitbox02-rust/src/workflow.rs

```
@@ -3,7 +3,10 @@
 pub mod cancel;
 pub mod confirm;
 pub mod menu;
-#[cfg_attr(feature = "c-unit-testing", path = "workflow/mnemonic_c_unit_tests.rs")]
+#[cfg_attr(
```

With `cfg_attr(all(feature = "c-unit-testing", not(feature = "testing")), ...)`, if someone (accidentally) enables both `c-unit-testing` and `testing` at the same time, the code silently uses the "real" `mnemonic.rs.` Would it be safer to add a `compile_error!` to catch this?